pyproject.toml

```diff
@@ -373,7 +373,14 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.cibuildwheel]
 # Skip unsupported platforms (by us or by Rust).
-skip = "cp36* *-musllinux_i686 pp*aarch64 *-musllinux_aarch64"
+# See https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip for the list of build targets.
+# We skip:
+#  - CPython 3.6 and 3.7: EOLed
+#  - PyPy 3.7: we only support Python 3.8+
+#  - musllinux i686: TODO why?
+#  - PyPy on Aarch64: TODO why?
+#  - musllinux on aarch64: TODO why?
```

Comment:
it seems related to speed of build according to https://github.com/matrix-org/synapse/pull/14259.

ISTR that aarch64 builds are slow as hell because they don't run natively but inside qemu, and the non CPython versions are a lot less useful to people.